.github/workflows/reactjs_test.yaml

```diff
@@ -0,0 +1,26 @@
+name: React.js CI/CD
+on:
+  pull_request:
```

Comment:
you should check test pipeline and try with your personal github action, change this line to on push spesific branch (for debug).

Reply:
then dont forget after all pipeline pass, change it again with pull_request trigger